src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -188,7 +188,7 @@
             @NotNull
             @PathVariable(value = "nomsNumber") final String nomsNumber) {
 
-        return getOffenderLatestRecall(offenderService.offenderIdOfNomsNumber(nomsNumber));
+        return new ResponseEntity<>(getOffenderLatestRecall(offenderService.offenderIdOfNomsNumber(nomsNumber)), HttpStatus.OK);
```

Comment:
Can we change the return type to drop the ResponseEntity?

Reply:
Ah I see - we don't need the ResponseEntity as we are only handling the happy path here.  Yep, will do.